wikipedia-sentences/README.md

```diff
@@ -66,59 +95,116 @@ shuf input.txt > input.txt
+|  |  |
+|--|--|
+| `top_k` | Integer determining the number of sentences to return |
+| `mode` | Mode to trigger in the call. See [here](https://docs.jina.ai/chapters/rest/) for more details |
+| `data` | Text input to query |
+ 
 
-### From the Terminal
+
+
+
+### Using Jina Box; our frontend search interface
+
+**Jina Box** is a light-weight, highly customizable JavaScript based front-end search interface. To use it for this example, begin by opening the REST API interface. 
+
+```sh
+python app.py -t query_restful
+```
+
+In your browser, open up the hosted JinaBox on [jina.ai/jinabox.js](https://jina.ai/jinabox.js/). In the configuration bar on the left hand side, choose a custom endpoint and enter the following: `http://127.0.0.1:45678/search` . You can type in search queries into the text box on the right hand side!
+
+
+### Directly from terminal
+You can also easily search (query) your data directly from the terminal. Using the following command will open an interface directly in your terminal window. 
 
 ```sh
 python app.py -t query
 ```
 
-## 👷 Build a Docker Image
+# 🔮 Overview of the files in this example
+Here is a small overview if you're interested in understanding what each file in this example is doing. 
+|File   | Explanation  |
+|--|--|
+|📂 `flows/`  | Folder to store Flow configuration    |
+|--- 📃 `flows/index.yml`  | Contains the details of which executors should be used for indexing your data. |
+|--- 📃 `flows/query.yml`  | Contains the details of which executors should be used for querying your data. |
+|📂 `pods/` | Folder to store Pod configurations|
+|--- 📃 `pods/encode.yml`  | Specifies the configurations values for the encoding executor.   |
+|--- 📃 `[pods/index.yml`  | Specifies the configurations values for the encoding executor.   |
+|📂 `test/*`  | Various maintenance tests to keep the example running.   |
+|📃 `app.py`   | The gateway code to combine the index and query flow.  |
+|📃 `get_data.sh`  |  Downloads the Kaggle dataset.|
+|📃 `manifest.yml`   |Needed to deploy to Jina Hub.|
+|📃 `requirements.txt`  |  Contains all required python libraries.|
 
-This will create a Docker image with pre-indexed data and an open port for REST queries.
 
-1. Run all the steps in setup and index first. Don't run anything in the search step!
-2. If you want to [push to Jina Hub](https://github.com/jina-ai/jina-hub) be sure to edit the `LABEL`s in `Dockerfile` and fields in `manifest.yml` to avoid clashing with other images
-3. Run `docker build -t <your_image_name> .` in the root directory of this repo
-5. Run it with `docker run -p 45678:45678 <your_image_name>`
-6. Search using instructions from Search section above
+# 🌀 Flow diagram
 
-### Image name format
+This diagram provides a visual representation of the two Flows in this example. Showing which executors are used in which order. 
```

Comment:
This diagram provides a visual representation of the two Flows in this example. Showing which executors are used in which order.

->

This diagram provides a visual representation of the two Flows in this example, showing which Executors are used in which order.

Reply:
fixed.